gcp-data-drive/getdata_test.go

```diff
@@ -11,62 +11,98 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
 package gcpdatadrive
 
 import (
 	"context"
-	"errors"
 	"net/http"
 	"reflect"
 	"testing"
 )
 
 var parseTests = []struct {
```

Comment:
Can just name these `tests`. Don't need the extra descriptive name when the scope is limited.

Reply:
fixed PHAL